src/ontology/mondo.Makefile

```diff
@@ -289,6 +289,46 @@ sources/$(SOURCE_VERSION)/equivalencies.owl: | source_release_dir
 #sources/CTD_diseases.obo:
 #	curl -L -s http://ctdbase.org/reports/CTD_diseases.obo.gz  | gzip -dc | perl -npe 's@alt_id@xref@' > $@.tmp && mv $@.tmp $@
 
+# TODO: Gard mapped obsoletes
+# Failed commands ------
+#assocs:  # not what I needed
+#	runoak -i sqlite:tmp/mondo-ingest.db associations -p i DOID:2757
+#axioms:  # notImplementedError
+#	runoak -i tmp/mondo-edit-temp.ofn axioms –axiom-type MONDO:equivalentTo
+#	runoak -i tmp/mondo-edit-temp.ofn axioms –axiom-type SubClassOf
+#relat:  # only shows subClassOf
+#	runoak -i sqlite:tmp/mondo-edit.db relationships -p i MONDO:0000004
+# #	runoak -i sqlite:tmp/mondo-edit.db relationships -p i MONDO:0000004 --predicates oboInOwl:hasDbXref skos:exactMatch skos:narrowMatch skos:broadMatch skos:relatedMatch
+
+# Working but not needed commands --------
+#tmp/mondo-edit.owl:
+#	robot convert -i mondo-edit.obo --format owl -o $@
+#ofn:
+#	robot convert -i mondo-edit.obo --format ofn -o tmp/mondo-edit.ofn
+
+# Working helper commands ------
+tmp/mondo.owl: tmp/mondo-version_current.owl
+	cp tmp/mondo-version_current.owl $@
+tmp/mondo.db: tmp/mondo.owl
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+tmp/mondo.sssom.tsv: tmp/mondo.db
+	runoak -i sqlite:tmp/mondo.db mappings -O sssom -o $@
+reports/obsoletes.tsv: tmp/mondo.db
+	runoak -i sqlite:tmp/mondo.db obsoletes > reports/obsoletes.tsv
+
+# todo: eventually
+# - maybe need to run the python script
+tmp/obsoletes_gard.tsv: reports/obsoletes.tsv tmp/mondo.sssom.tsv
```

Comment:
@matentzn I'm a bit baffled. Any idea why when I run this, it always also runs `tmp/mondo.sssom.tsv`? That's even if `tmp/mondo.sssom.tsv` and all of the dependencies in its prerequisite goal chain already exist as well.

Note that if I change `tmp/mondo.sssom.tsv: tmp/mondo.db` to `tmp/mondo.sssom.tsv:`, suddenly it solves this problem, but I don't know why.

Here's the dependency tree.
```make
tmp/mondo-version_current.owl:
	$(ROBOT) merge -I $(OBO)/mondo.owl -o $@

tmp/mondo.owl: tmp/mondo-version_current.owl
	cp tmp/mondo-version_current.owl $@
tmp/mondo.db: tmp/mondo.owl
	@rm -f .template.db
	@rm -f .template.db.tmp
	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
	@rm -f .template.db
	@rm -f .template.db.tmp
tmp/mondo.sssom.tsv: tmp/mondo.db
	runoak -i sqlite:tmp/mondo.db mappings -O sssom -o $@
tmp/obsoletes_gard.tsv: reports/obsoletes.tsv tmp/mondo.sssom.tsv
	python temp_mondo_gard_obsoletes.py
```

Note that I tried removing prerequisites from `tmp/mondo.db` and `tmp/mondo.owl` goals as well to see if that would stop this, but that didn't work.Only removing the prerequisite from `tmp/mondo.sssom.tsv` goal worked.

Reply:
The problem was that I didn't know how to locate `mondo.owl`. All I needed was `$(ONT).owl`